src/cli/index.ts

```diff
@@ -53,6 +53,10 @@ function progress({ event, script, updatedFiles, skippedFiles, newVersion }: Ver
       console.log(info, `${skippedFiles.pop()} did not need to be updated`);
       break;
 
+    case ProgressEvent.ScriptSkipped:
```

Comment:
No need to print a message for skipped scripts.  If they set the `--no-scripts` option, then we just won't print anything about the scripts, same as if they don't have scripts in their `package.json` file